controls/package_spec.rb

```diff
@@ -101,3 +101,73 @@
     its('disk_error_action') { should cmp 'SUSPEND' }
   end
 end
+
+control 'package-09' do
+  impact 1.0
+  title 'Install packages suggested by Lynis'
```

Comment:
It really feels that we need to create something like a best-practice baseline for os operations since those are not focussing on hardening. @artem-sidorenko @atomic111 eg. linux-ops-baseline, we could put this in incubation mode to collect best-practices there.